1.1/validator.ttl

```diff
@@ -21,22 +21,31 @@ As of GeoSPARQL 1.1, this validator is not normaltive, only informative, however
 	] ;
 	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
 	dcterms:created "2021-06-13"^^xsd:date ;
-	dcterms:modified "2021-06-13"^^xsd:date ;
+	dcterms:modified "2021-06-22"^^xsd:date ;
 	owl:versionInfo "OGC GeoSPARQL 1.1" ;
 	owl:versionIRI <http://www.opengis.net/def/geosparql/validator/1.1> ;  # same as Spec version for now
 .
 
-<feature-hasGeometry>
+<hasGeometry-hasSerialization>
     a sh:NodeShape ;
-	sh:targetClass geo:Feature ;
+	sh:targetSubjectsOf geo:hasGeometry ;
```

Comment:
targetObjectsOf

Reply:
Fixed in Commit https://github.com/opengeospatial/ogc-geosparql/pull/152/commits/747ee942174a35a8f3e2d676b0b4671d4c86fd7f